doc/source/ray-observability/ray-distributed-debugger.rst

```diff
@@ -77,6 +77,50 @@ Start a Ray cluster
     After checking that `ssh -p 2222 ray@localhost` works, set up VS Code as described in the
     `VS Code SSH documentation <https://code.visualstudio.com/docs/remote/ssh>`_.
 
+  .. tab-item:: KubeRay (Code Server)
```

Comment:
Can you add a note in this tab that this approach and container is community maintained?